src/main/java/bio/terra/cli/context/GlobalContext.java

```diff
@@ -153,7 +154,8 @@ public void updateDockerImageId(String dockerImageId) {
   // Directory and file names
   //   - top-level directory: $HOME/.terra
   //   - persisted global context file: global-context.json
-  //   - sub-directory for persisting pet SA keys: pet-keys
+  //   - sub-directory for persisting pet SA keys: pet-keys/[terra user id]
+  //   - pet SA key filename: [workspace id]
```

Comment:
nit: maybe indent `- pet SA key filename` to make it clear it's under the sub-directory?

Reply:
good idea. done.